src/site/_includes/components/AuthorCard.js

```diff
@@ -23,6 +23,7 @@ const BaseCard = require('./BaseCard');
  */
 class AuthorCard extends BaseCard {
   constructor(arg) {
+    arg.collectionItem = arg.author;
```

Comment:
mentioned in Chat. Let's explore just getting rid of the named arguments and make BaseCard take three arguments instead of an object